applications/fishing-map/src/data/config.ts

```diff
@@ -46,3 +49,45 @@ export const sources: SelectOption[] = [
 ]
 
 export const DATE_FORMAT = 'yyyy/MM/dd'
+
+export const timebarEventsOptions: SelectOption[] = [
+  {
+    id: 'all',
+    label: 'All events',
+  },
+  {
+    id: 'fishing',
+    label: 'Fishing',
+  },
+  {
+    id: 'encounters',
+    label: 'Encounters',
+  },
+  {
+    id: 'loitering',
+    label: 'Loitering',
+  },
+  {
+    id: 'ports',
+    label: 'Port visits',
+  },
+  {
+    id: 'none',
+    label: 'None',
+  },
+]
+
+export const timebarGraphOptions: SelectOption[] = [
```

Comment:
```suggestion
export const TIMEBAR_GRAPH_OPTIONS: SelectOption[] = [
```


Please use SCREAMING_SNAKE_CASE by convention when setting readonly consts